### ISSUE TITLE
Stolen Project 

### ISSUE THREAD

**Issue by UmairFaheem** [naveens852001/progresspulse#1]
This project is a complete copy of the work done by me. This person just came out of nowhere and stole the project. 

I made this project public to show my project and what I invested in it.

Nevertheless I can make another one and then another one. You stick to stealing because that's what you do the best.

Original repository: https://github.com/UmairFaheem042/MERN-ProBoard-Project_Management_Tool

**Comment by mujahidh22** [comment]
Agreed with UmairFaheem. I had checked his repo and the timeline for his project is earlier than this guy. He's just copy pasted the code.
